python/sglang/srt/model_executor/model_runner.py

```diff
@@ -895,6 +897,14 @@ def init_memory_pool(
                 )
             self.max_total_num_tokens = min(self.max_total_num_tokens, max_total_tokens)
 
+        # modify max_total_num_tokens due to ReqToTokenPool
+        self.max_total_num_tokens -= (
```

Comment:
I think we should move this subtraction to `profile_max_num_token`. Our previous profile was not accurate